pkg/client/cache/eventqueue.go

```diff
@@ -36,12 +36,13 @@ import (
 // watch.Deleted event (though this state is delivered by the watch API itself, it is not passed on
 // to the reflector Store).
 type EventQueue struct {
```

Comment:
Looks like the SDN and the router are the only things that use this.  I think we should see if we can collapse this onto more "standard" infrastructure now that we've developed constructs that may serve: framework.Informer, workqueue, DeltaFIFO.